src/ui/FacetSlider/FacetSlider.ts

```diff
@@ -510,10 +511,6 @@ export class FacetSlider extends Component {
     this.delayedGraphData = null;
   }
 
-  public isSimpleSliderConfig() {
```

Comment:
If you add a `get` to this method, with the same logic/the logic on line 386, wouldn't you get the same functionality but with the advantage that the value cannot be overwritten?

Reply:
No, options are public, so other code could go in and do something like `facetSlider.options.start = 12345;`, which would mess up the `get` method.

By setting it initially at the start of the constructor we're preventing that.